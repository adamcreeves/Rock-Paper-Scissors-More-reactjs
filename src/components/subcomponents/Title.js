import React from "react";
import "../../styles/Main.css";

function Title({ str, classNm }) {
  return (
    <div className={classNm} data-testid={"title-component"}>
      {str}
    </div>
  );
}

export default Title;
