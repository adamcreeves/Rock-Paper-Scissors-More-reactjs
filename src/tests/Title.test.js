import React from "react";
import { render } from "@testing-library/react";
import Title from "../components/subcomponents/Title";

describe("Test Body", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Title />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("should render Body and not be null", () => {
    const { asFragment } = render(<Title />);
    expect(asFragment()).not.toBeNull();
  });
});
