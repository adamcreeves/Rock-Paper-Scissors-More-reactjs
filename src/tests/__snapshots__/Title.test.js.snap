// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Body should match the snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="title-component"
  />
</DocumentFragment>
`;
